### DISCUSSION TITLE
Fail2ban logging when Kimai runs in docker

### DISCUSSION THREAD

**Discussion by BeckeBauer** [kimai/kimai#4823] [Plugins]
After I run Kimai in Docker, the recorded fail2ban message changed as follows: 
`[2024-04-30T10:22:30.805493+00:00] fail2ban.ERROR: HOST-IP [] {"channel":"fail2ban"}`
The IP of the client trying and failing to access is not recorded but the docker host IP. Any chance to update the plugin to be compatible with Docker? 

Maybe, the issue is related to this discussion: [#431](https://github.com/tobybatch/kimai2/discussions/431) ?

**Comment by mplawner** [comment]
Not sure if you got this resolved. 
I was able to get it working, but the solution isn't persistent. I've opened a discussion thread here:
https://github.com/kimai/kimai/discussions/5020

If you had any additional luck in getting this to work, please share. Thanks!

**Reply by BeckeBauer** [reply]
No solution so far - was not even able to reproduce your work around (I am using the kimai2:fpm-prod image)

**Reply by kevinpapst** [reply]
@BeckeBauer please read the docs, you are on a deprecated tag https://www.kimai.org/documentation/docker.html

**Reply by BeckeBauer** [reply]
oops, thank you for pointing out the obvious (I read about the changes without realising the need for action)
